### PR TITLE
allow to reject the job and case in accepted state

### PR DESCRIPTION
**Pull request by DmytroHryshyn**

### CODE REVIEW

File: src/components/fileService.ts [hunk 1]
```diff
@@ -64,7 +52,7 @@ export class FileService {
 
 		this.#messageBus.publish({
 			kind: MessageKind.deleteFiles,
-			uris: [message.oldUri, message.newContentUri],
```
Comment (grzpab) [inline src/components/fileService.ts:67]: what does this change do?

File: src/components/jobManager.ts [hunk 2]
```diff
@@ -330,7 +330,10 @@ export class JobManager {
 			if (
 				job &&
 				(job.kind === JobKind.rewriteFile ||
-					job.kind === JobKind.moveAndRewriteFile) &&
+					job.kind === JobKind.moveAndRewriteFile ||
```
Comment (grzpab) [inline src/components/jobManager.ts:333]: can you explain this change?

File: src/components/fileService.ts (hunk 1 above)
Comment (DmytroHryshyn) [reply src/components/fileService.ts:67]: We dont want to remove codemod output files from storage immediately after changes are applied, so i've removed message.newContentUri from that array.

File: src/components/jobManager.ts (hunk 2 above)
Comment (DmytroHryshyn) [reply src/components/jobManager.ts:333]: This should clean all output files of the job from storage when the job is rejected

**Review by grzpab — APPROVED** [review]

**Pull request merged**